packages/ui/components/ui/tx/view/action-dutch-auction-schedule.tsx

```diff
@@ -0,0 +1,63 @@
+import { ActionDutchAuctionSchedule } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/auction/v1alpha1/auction_pb';
+import { ViewBox } from './viewbox';
+import { ActionDetails } from './action-details';
+import {
+  AssetId,
+  ValueView,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
+import { ValueViewComponent } from './value';
+import { Amount } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/num/v1/num_pb';
+
+const getValueView = (amount?: Amount, assetId?: AssetId) =>
+  new ValueView({
+    valueView: {
+      case: 'knownAssetId',
+      value: {
+        amount,
+        metadata: {
+          penumbraAssetId: assetId,
+        },
+      },
+    },
+  });
+
+export const ActionDutchAuctionScheduleComponent = ({
```

Comment:
![image](https://github.com/penumbra-zone/web/assets/1121544/fb395d7d-ba84-4180-a2bb-55178e4062c4)

Reply:
At the moment, auction-related actions are represented in a `TransactionView` as the action objects themselves not as `*View` objects. So we don't currently have metadata to render the assets with.

This should hopefully be fixed soon.

Reply:
Good call — fixed